internal/cephes/cephes.go

```diff
@@ -21,7 +21,8 @@ var (
 )
 
 const (
-	machEp = 1.11022302462515654042e-16 // 2^-53
-	maxLog = 7.09782712893383996732e2   // log(2^127)
-	minLog = -7.451332191019412076235e2 // log(2^-128)
+	machEp  = 1.11022302462515654042e-16 // 2^-53
```

Comment:
I decided to go ahead with the change since it was small and isolated. I used your `machEp` suggestion but for the other two I used constants from the math package.